services/sidecar/src/simcore_service_sidecar/mpi_lock.py

```diff
@@ -93,9 +96,9 @@ def is_locked_factory():
 
     if not is_lock_free:
         # it was not possible to acquire the lock
```

Comment:
Why not setting here already   ``reply_queue.put(False)``

Reply:
To be sure the there will always be a result in the queue `_acquire_and_extend_lock_forever` will always put a False inside for a response.